proposals/lip-0054.md

```diff
@@ -113,15 +113,10 @@ Furthermore, for the rest of this section:
 We also define the following auxiliary function to make the specifications more readable.
 
 ```python
-def emitCCMEvent(ccm: CCM, result: uint32, code: uint32) -> None:
-    emitEvent(
-        module = MODULE_NAME_INTEROPERABILITY,
-        name = EVENT_NAME_CCM_PROCESSED,
-        data = {"ccm": ccm, "result": result, "code": code},
-        topics = [ccm.sendingChainID, ccm.receivingChainID]
-    )
 ```
 
+We also make use of the [`emitCCMEvent` function][[lip-0045#emitccmevent]] defined in [LIP 0045][lip-0045].
```

Comment:
```suggestion
We also make use of the [`emitCCMEvent` function][lip-0045#emitccmevent] defined in [LIP 0045][lip-0045].
```

Reply:
This was fixed in my local machine but forgot to push, should be ok now.